src/modes/shop/shop.cpp

```diff
@@ -1933,14 +2003,17 @@ void ShopMode::RemoveObjectToSell(uint32_t object_id)
 
 void ShopMode::RemoveObjectToTrade(uint32_t object_id)
 {
-    std::map<uint32_t, ShopObject *>::iterator shop_iter = _available_trade.find(object_id);
+    std::map<uint32_t, ShopObject *>::iterator shop_iter =
+      _available_trade.find(object_id);
```

Comment:
When splitting a line like this, could you use 4 spaces as well, except for if or other condition alignment?